### ISSUE TITLE
Problem loading tampers

### ISSUE THREAD

**Issue by naoxide95** [ron190/jsql-injection#94688]
I am testing a vulnerable URL and trying to use tampers attached with the tool. Whenever I check some tamper options from the settings, I keep getting this error message:

`[13:21:26,897] Tampering context contains errors: TypeError: Cannot load script from ../model/src/main/resources/tamper/js/random-case.js in <eval> at line number 1`

I'm using jsql v0.84 on windows server 2019, with java jdk 13.0.2

**Comment by ron190** [comment]
Reproduced using v0.84 on jdk13, thanks for the feedback.

It appears that only the tamper script `Random SQL characters case` fails, other scripts are looking fine to me.

So the `Random char case` uses `load()` with an incorrect path, you can see the wrong path when you mouse over this specific tamper script in jSQL.

Here's your choice:

- Copy/paste the fixed script in custom tamper field in jSQL (see screenshot), enable the checkbox and run again:
```javascript
load('classpath:tamper/js/random-case.js'); // <-- Fixed path

var tampering = function(sql) {
   
    // Name of database and table is case sensitive on Linux (column name is case insensitive)
    // Script js/random-case.js ignore case for expression `database`.`table`
    return randomCase(sql)
        
        // Fix hexa prefix
        .replace(/0X/g,'0x');
}
```
- Or just wait for a fix that will come with v0.85

<a href=https://user-images.githubusercontent.com/1971480/115446726-eaabb080-a217-11eb-843b-c960d3f6626d.png><img width=400 src=https://user-images.githubusercontent.com/1971480/115446726-eaabb080-a217-11eb-843b-c960d3f6626d.png /></a>

**Comment by naoxide95** [comment]
Thank you!
I was able to override the problem using the first proposed solution, and I hope issue will be fixed in the next update.
